pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -2614,3 +2616,34 @@ func validatePersistentState(field *k8sfield.Path, spec *v1.VirtualMachineInstan
 
 	return
 }
+
+func validateFileMemoryBacking(field *k8sfield.Path, spec *v1.VirtualMachineInstanceSpec) (causes []metav1.StatusCause) {
+	if !utils.IsFileMemoryBackedVmi(&v1.VirtualMachineInstance{Spec: *spec}) {
+		return
+	}
+
+	getField := func() string {
+		return field.Child("domain", "memory", "backed", "file").String()
+	}
+
+	if utils.IsHighPerformanceVmi(&v1.VirtualMachineInstance{Spec: *spec}) {
+		causes = append(causes, metav1.StatusCause{
+			Type:    metav1.CauseTypeFieldValueNotSupported,
+			Message: "Memory file is supported only for non-high-performance VMIs",
+			Field:   getField(),
+		})
+	}
+
+	for _, disk := range spec.Domain.Devices.Disks {
```

Comment:
we could default to `CacheNone` in the converter if there is no cache set. The user then doesn't need to explicitly opt out from `Cache`. WDYT?

Reply:
I think it's a good idea